requirements.txt

```diff
@@ -59,6 +59,6 @@ distlib~=0.3.6
 Empty~=0.4.4
 cachetools~=5.3.0
 tomlkit~=0.11.8
```

Comment:
Do we need these all libraries for the SDK to run @sunil-lakshman

Reply:
We are using above packages @ishaileshmishra 
1. **Empty** - is a light wrapper around Flask which adds some sensitive defaults to your project out of the box.
2. **tomlkit**  -  It includes a parser that preserves all comments, indentations, whitespace and internal element ordering, and makes them accessible and editable via an intuitive API.
3. **cachetools** - For our purpose, a cache is a key-value store, aka a dictionary, possibly with some constraints on keys (type, size) and values (size, serialization). Dependency for **logging** package.

Reply:
@sunil-lakshman What if you remove those and run tests. Does tests fail ?

Reply:
yes @ishaileshmishra 
![Uploading Screenshot 2023-11-01 at 5.10.39 PM.png…]()

Reply:
I doubt about Empty and cachetools. Are they really required ?

Reply:
Getting above error if i remove Empty from code.

Reply:
okay got it, what if we remove cachetools only ?